### PR TITLE
Userstory11

### PR DESCRIPTION
**Pull request by kornrumpfl**
> input to update date and time on the flight plan page when data is loaded from saved flight plans.
> control so that the flight plan is not saved in duplicate and to update the flight plan that's been changed.
> saved flight plans to show date and time.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -35,22 +35,30 @@ function App() {
...
-        flightTime,
-        aircraft,
-        numberOfPassengers,
-        departureLocation,
-        arrivalLocation,
-      },
-    ]);
+    if (savedFlightPlanData.find((item) => id === item.id)) {
+      const myId = savedFlightPlanData.findIndex((item) => item.id === id);
+      console.log(myId);
```
Comment (jecaps) [inline src/App.js]: I don't think you need the console.log here :)

File: src/components/flightPlan/flightPlan.js [hunk 2]
```diff
@@ -98,28 +101,32 @@ export default function FlightPlan({ flightPlanData, onSavePlan }) {
...
-            </p>
+            <input
+              id="flightDateFP"
+              type="date"
+              defaultValue={flightPlanData.flightDate}
+              min={new Date().toISOString().split("T")[0]}
+              onChange={(event) => setFlightDate(event.target.value)}
+            />
+            <input
+              id="flightTimefP"
```
Comment (jecaps) [inline src/components/flightPlan/flightPlan.js]: Should this be`flightTimeFP`? Just asking cuz its `flightDateFP` on the date input. Just to be uniform :)

File: src/components/saved/SavedFlightPlans.js [hunk 3]
```diff
@@ -100,3 +107,26 @@ const SavedItem = styled.li`
...
+  p {
+    margin-bottom: 2px;
+  }
+`;
+
+const Block = styled.div`
+  display: flex;
+  flex-direction: column;
+  span {
+    margin-top: 0px;
```
Comment (jecaps) [inline src/components/saved/SavedFlightPlans.js]: You can get rid of the `px` on 0 measurements. :)

**Review by jecaps — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (kornrumpfl) [reply src/App.js]: This was for testing! Removed! Thanks

File: src/components/flightPlan/flightPlan.js (hunk 2 above)
Comment (kornrumpfl) [reply src/components/flightPlan/flightPlan.js]: Done

File: src/components/saved/SavedFlightPlans.js (hunk 3 above)
Comment (kornrumpfl) [reply src/components/saved/SavedFlightPlans.js]: Thanks, forget to remove it after testing!

**Review by jecaps — APPROVED** [review]

**Pull request merged**